CMakeLists.txt

```diff
@@ -169,6 +170,9 @@ install(
     LIBRARY
         DESTINATION
             ${CMAKE_INSTALL_LIBDIR}
+    RUNTIME
+        DESTINATION
+            ${CMAKE_INSTALL_BINDIR}
```

Comment:
Why `CMAKE_INSTALL_BINDIR` rather than `CMAKE_INSTALL_LIBDIR`?

Reply:
According the CMake docs(https://cmake.org/cmake/help/latest/command/install.html):
>Executables are treated as RUNTIME objects, except those marked with the MACOSX_BUNDLE property on macOS (see BUNDLE below.) For DLL platforms (all Windows-based systems including Cygwin), the DLL part of a shared library is treated as a RUNTIME target.

Since executables(like `uriparse`) are installed to `bin`, there is no reason to install the DLLs to lib directory. Otherwise, executables will not find the DLLs they relay on.

Installing the `RUNTIME` to `CMAKE_INSTALL_BINDIR` is taken by many widely used projects:
https://github.com/edenhill/librdkafka/blob/master/src/CMakeLists.txt#L288
https://github.com/QuantStack/xeus/blob/master/CMakeLists.txt#L312
https://github.com/zeromq/libzmq/blob/master/CMakeLists.txt#L1367

Reply:
@hartwork Sure. Thanks!

Reply:
Marged, thank you!